protocol/triple/server.go

```diff
@@ -144,6 +147,44 @@ func (s *Server) Start(invoker protocol.Invoker, info *server.ServiceInfo) {
 	}()
 }
 
+// RefreshService refreshes Triple Service
+func (s *Server) RefreshService(invoker protocol.Invoker, info *server.ServiceInfo) {
```

Comment:
Maybe we could define a common function to initialize ```hanOpts``` since there are much repeated logic.